### DISCUSSION TITLE
Idea per l'utente coordinatore

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#289] [Idee e proposte]
Il coordinatore può aver accesso alle altre discipline dall'area coordinatore per modificare voti, compiti ecc

**Comment by trinko** [comment]
> Il coordinatore può aver accesso alle altre discipline dall'area coordinatore per modificare voti, compiti ecc

Questo assolutamente no. Non è pensabile che il coordinatore (ma neanche il dirigente) possa modificare voti o altro inserito da un docente: sarebbe del tutto illeggittimo, lesivo della professionalità del docente e fonte di contenziosi.

**Reply by manuelbarberi** [reply]
L'ho chiesto dato che su axios è presente questa funzionalità... anche solo per vedere i voti.

**Comment by trinko** [comment]
Per vedere tutti i voti dei docenti si, è una cosa che conto di introdurre. Anche la visione dell'agenda con le verifiche previste da tutti i docenti.
La segno come richiesta.
